.python-version

```diff
@@ -0,0 +1 @@
+3.6.11
```

Comment:
What is purpose of this file? Is it needed in PR?

Reply:
nope that's an accidental add because of pyenv.

Reply:
I need it against 4.x because I'm not ready to move to celery 5